content/docs/concepts/traffic-management/network/index.md

```diff
@@ -0,0 +1,320 @@
+{{< /warning >}}
+
+## Timeouts and retries
+
+You can use Istio's traffic management resources to set defaults for timeouts
+and retries per service and subset that apply to all callers.
+
+### Override default timeout setting
+
+The default timeout for HTTP requests is 15 seconds. You can configure a
+virtual service with a routing rule to override the default, for example:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+    - ratings
+  http:
+  - route:
+    - destination:
+        host: ratings
+        subset: v1
+    timeout: 10s
+{{< /text >}}
+
+### Set number and timeouts for retries
+
+You can specify the maximum number of retries for an HTTP request in a virtual
+service, and you can provide specific timeouts for the retries to ensure that
+the calling service gets a response, either success or failure, within a
+predictable time frame.
+
+Envoy proxies automatically add variable jitter between your retries to
+minimize the potential impact of retries on an overloaded upstream service.
+
+The following virtual service configures three attempts with a 2-second
+timeout:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+    - ratings
+  http:
+  - route:
+    - destination:
+        host: ratings
+        subset: v1
+    retries:
+      attempts: 3
+      perTryTimeout: 2s
+{{< /text >}}
+
+Consumers of a service can also override timeout and retry defaults with
+request-level overrides through special HTTP headers. The Envoy proxy
+implementation makes the following headers available:
+
+-  Timeouts: `x-envoy-upstream-rq-timeout-ms`
+
+-  Retries: `X-envoy-max-retries`
+
+## Circuit breakers
+
+As with timeouts and retries, you can configure a circuit breaker pattern
+without changing your services. While retries let your application recover from
+transient errors, a circuit breaker pattern prevents your application from
+stalling as it waits for an upstream service to respond. By configuring a
+circuit breaker pattern, you allow your application to fail fast and handle the
+error appropriately, for example, by triggering an alert. You can configure a
+simple circuit breaker pattern based on a number of conditions such as
+connection and request limits.
+
+### Limit connections to 100
+
+The following destination rule sets a limit of 100 connections for the
+`reviews` service workloads of the v1 subset:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews
+spec:
+  host: reviews
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+    trafficPolicy:
+      connectionPool:
+        tcp:
+          maxConnections: 100
+{{< /text >}}
+
+See the [circuit-breaking task](/docs/tasks/traffic-management/circuit-breaking/)
+for detailed instructions on how to configure a circuit breaker pattern.
+
+## Health checking
```

Comment:
Please call this as Outlier Detection, as it aligns well with the concept in the destination rule

Reply:
Actually no.. Please remove this entire section. We dont support this feature at all. Not what you are talking about.

Reply:
@frankbu @andraxylia Do you concur?

Reply:
Even though it mentions that active health checking is not supported, this whole section is describing active health checking. As Shriram said, Istio doesn't do that at all, so removing the whole section would be best.